msgraphcore/middleware/options/middleware_control.py

```diff
@@ -18,10 +17,12 @@ def get_middleware_options(self, func):
 
         def wrapper(*args, **kwargs):
             # Get middleware options from **kwargs
-            scopes = kwargs.pop('scopes', None)
-            if scopes:
-                # Set middleware options, for use by middleware in the middleware pipeline
-                self.set(AUTH_MIDDLEWARE_OPTIONS, AuthMiddlewareOptions(scopes))
+            middleware_options = kwargs.pop('middleware_options', None)
```

Comment:
Why are we making this change?

Reply:
Removed as we will not be using an options bag.